SwiftyStoreKit/InAppReceipt.swift

```diff
@@ -168,6 +168,39 @@ internal class InAppReceipt {
             return .expired(expiryDate: firstExpiryDateItemPair.0, items: sortedReceiptItems)
         }
     }
+    
+    /**
+     *  Get the distintc product identifiers from receipt.
```

Comment:
There is a typo `distintc` to `distinct`

Reply:
Fixed, thanks for review :)